libs/passport.js

```diff
@@ -65,82 +65,86 @@ var drupal_login = function (email, password, done) {
         drupal_login_request(email, password).then(function (drupal_user) {
             if (drupal_user != null) {
                 // Drupal update information.
-                var drupal_user_id = drupal_user.user.uid;
-                var tickets = drupal_user.user.data.tickets.tickets;
-                var current_event_tickets_count = 0;
-                var tickets_array = [];
+                var drupal_user_id = _.get(drupal_user, 'user.uid');
+                if (!drupal_user_id) {
+                    done(false, i18next.t('invalid_user_id'));
```

Comment:
This is the only change right, all the rest is indentation?

Reply:
mostly yes, you can see only the changes without whitespace changes by adding `?w=1` to the url - https://github.com/Midburn/spark/pull/577/files?w=1

Reply:
Thanks! it's useful.